broker/service/spreadsheet_upload_service.py

```diff
@@ -18,7 +18,12 @@ def __init__(self, ingest_api: IngestApi, storage_service: SpreadsheetStorageSer
         self.storage_service = storage_service
         self.importer = importer
 
-    def async_upload(self, token, request_file, is_update, project_uuid=None, submission_uuid=None):
+    def async_upload(self, token, request_file, params):
+        project_uuid = params.get('projectUuid')
+        submission_uuid = params.get('submissionUuid')
+        is_update = params.get('isUpdate')
```

Comment:
What happens if these parameters are not provided in the `params` dictionary? Previously `is_update` did not have a default value.

Reply:
It's okay for all these params to be None. They are all optional. In line 34 below, the `if is_update` condition will just evaluate to false.

Reply:
The flag is_update is for importing a spreadsheet with UUIDs and updates. If it's falsy like None, it means it's a new submission and `project_uuid` and `update_project` will indicate that there is an existing project and the user preference to update it or not.